sotoki/sotoki.py

```diff
@@ -879,30 +889,64 @@ def get_filetype(headers, path):
             ftype = "ico"
     return ftype
 
+def download_from_cache(key, output, size, cache_storage):
+    ret = False
+    if cache_storage.has_object_matching_meta(key, "size", str(size)):
+        try:
+            cache_storage.download_file(key, output, progress=True)
+            print("Successfully downloaded already optimized version from cache.")
```

Comment:
Remove print

Reply:
I think we should inform of downloading status from cache. It might be helpful if we run into some bug in the future.

Reply:
we should print on failure but since we don't print on regular downloads, no need to print on successful downloads from cache.
Take a look at youtube merge commit https://github.com/openzim/youtube/commit/8bdc41af4afe102c6be0b57836ed8e0d8a548288 there's a lot of those decisions already sorted out. Might help you decide on some of those.